cpp/frameProcessor/src/SharedMemoryController.cpp

```diff
@@ -174,23 +163,14 @@ void SharedMemoryController::handleRxChannel()
 
       int bufferID = rxMsg.get_param<int>("buffer_id", -1);
       if (bufferID != -1) {
-
         if (sbm_) {
-
           // Create a frame object and copy in the raw frame data
           int frame_number = rxMsg.get_param<int>("frame", 0);
 
-          FrameProcessor::FrameMetaData frame_meta(frame_number,
-                                                    "raw",
-                                                    FrameProcessor::raw_64bit,
-                                                    "",
-                                                    std::vector<unsigned long long>());
+          FrameProcessor::FrameMetaData frame_meta(frame_number, "raw", FrameProcessor::raw_64bit, "", std::vector<unsigned long long>());
```

Comment:
Really long lines can still be split over multiple lines, but at the next indent level, not the arbitrary point where `(` is on the previous line.

```suggestion
          FrameProcessor::FrameMetaData frame_meta(
            frame_number, "raw", FrameProcessor::raw_64bit, "", std::vector<unsigned long long>()
          );
```

If it is still too long (this one isn't) each parameter can be split, e.g.

```suggestion
          FrameProcessor::FrameMetaData frame_meta(
            frame_number,
            "raw",
            FrameProcessor::raw_64bit,
            "",
            std::vector<unsigned long long>()
          );
```

Reply:
Thanks for the comment Gary, I modified another line of code which I deemed a bit long, could you verify the formatting too?